src/ui/MLFacet/MLFacet.ts

```diff
@@ -366,9 +387,9 @@ export class MLFacet extends Component {
     this.updateAppearance();
   }
 
-  private onQueryResponse(values: IFacetResponseValue[]) {
+  private onQueryResponse(values: IFacetResponseValue[], moreValuesAvailable: boolean) {
     this.header.hideLoading();
-    this.values.createFromResults(values);
+    this.values.createFromResults(values, moreValuesAvailable);
```

Comment:
Similarly here, would it be better to pass the response? If so, renaming to `createResultsFromResponse` might also be good.